Dockerfile

```diff
@@ -1,12 +1,15 @@
-FROM silintl/php7:7.2
+FROM silintl/php7:7.4
 
 MAINTAINER Phillip Shipley <phillip.shipley@gmail.com>
 
-ENV REFRESHED_AT 2020-04-16
+ENV REFRESHED_AT 2020-06-14
```

Comment:
2021?  :-) 
(I wonder if we missed that on other PR's, but I guess it may not matter too much.)

Reply:
Good catch 🙂